shell.nix

```diff
@@ -67,6 +67,18 @@ let
     rm $out/wallet-topology.yaml
     cp ${demoTopologyYaml} $out/wallet-topology.yaml
   '';
+  buildShell = pkgs.stdenv.mkDerivation {
```

Comment:
Maybe add a comment about what this is -- I suppose it's everything required for the `build-installer-unix.sh` script. Does it need the `autoStartBackend` conditional?